controllers/storagecluster/storageconsumer.go

```diff
@@ -294,9 +312,17 @@ func getLocalVolumeSnapshotClassNames(ctx context.Context, kubeClient client.Cli
 
 	vscSpec := make([]ocsv1a1.VolumeSnapshotClassSpec, 0, len(volumeSnapshotClassNames))
 	for vscName := range maps.Keys(volumeSnapshotClassNames) {
-		vscSpec = append(vscSpec, ocsv1a1.VolumeSnapshotClassSpec{Name: vscName})
+		vscSpec = append(
+			vscSpec,
+			ocsv1a1.VolumeSnapshotClassSpec{
+				CommonClassSpec: ocsv1a1.CommonClassSpec{
+					Name: vscName,
+				},
+			},
+		)
```

Comment:
There is no point to inline if the inlining makes the code blot. It makes the code harder to read 

```suggestion
		vscItem = ocsv1a1.VolumeSnapshotClassSpec{}
		vscItem.Name = vscName
		vscSpec = append(vscSpec, vscItem)
```

The same should be applied in other place where we have a similar pattern.

Reply:
done.